### ISSUE TITLE
Touch / Gesture bugginess with Appsee

### ISSUE THREAD

**Issue by patrickdevivo** [tomauty/react-native-appsee#2]
Not sure if this is the right place to post this, I've reached out to Appsee support.

I am using this Appsee React Native integration. I've installed / linked the library fine according to the instructions on Appsee and am able to send data into Appsee fine. However, I've noticed that when I import the library into my JS code, gestures and taps seem to be a bit mangled in my app. The biggest problem is I am unable to fully swipe two side menus I have into view. I begin sliding, but then the menu gets stuck, almost as if the code responding to the gesture has frozen. Another problem I've noticed is that button / UI taps often take two or three tries before registering, whereas they were very responsive before importing the library.

If I comment out the import statement in my JS code, everything works as is it should. As soon as I import the library however, I get all the bugginess again.

I was wondering if anyone else has experienced anything like this and if there are any known tricks to importing the library more smoothly.

I should note, when I install the iOS library manually (not through this package), I experience the same issue. Somehow Appsee is interfering with React Native?

**Comment by martinbooth** [comment]
I have experienced the same issue when using Appsee. I have seen that anything that should ordinarily respond to a tap inside a scrollview ignores the first tap immediately after the scrollview has scrolled.

Would be great if you can share any response you received from Appsee on this

**Comment by tomauty** [comment]
Haha this is weird. I made this repo a while ago and never published my integration. Meanwhile, they made one themselves. I've had this EXACT problem on iOS recently and it's been driving me INSANE. Just re-integrated them. Thank you so much.... I'll investigate.

**Comment by ylac** [comment]
Yeah I'm having the same issue with swiping left/right on Flatlist rows using the Swipeable module and the official react-native-appsee module. The swipe can only go a little bit before stopping.

**Comment by salujaharkirat** [comment]
I am facing the same issue in which navigator transitions become really slow on iOS.

I debugged this further and found that UI frame rates were consistently around 40-42 and with heavy data calls it went down to 15-20 and never came up. This was really messing up with navigations.

Commented out the appsee code and it works fine. Is there any solution to this?
